code/game/objects/structures/banking/banking.dm

```diff
@@ -0,0 +1,223 @@
+/**
+ * Everything to do with international banking
+ * 
+ * SPONSORED by the Foreign Nations Alliance (FNA)!
+ */
+
+/obj/structure
+	icon = ''
+	w_class = 10
+
+	var/climbable = FALSE
+	var/breakable = FALSE
+	var/parts
+	var/list/climbers = list()
+	var/low = FALSE
+
+
+	var/not_movable = TRUE
+	var/not_disassemblable = TRUE
+
+/obj/structure/banking
+    name = "banking"
+    desc = "debug"
```

Comment:
You don't need to make a debug version as it probably just breaks things, define all of the things you need in banking/computer/atm.

the vars could go under there too

Reply:
atleast that's how I see our code just going off structure/[x] and making things.